### PR TITLE
Surrogate Keys post - elaborate on null value logic

### PR DESCRIPTION
**Pull request by patkearns10**
## Description & motivation

Adding more color to this post about why this is important. 
Slack thread:  https://dbt-labs.slack.com/archives/C02G5V9JBHV/p1638572036249800

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

### CODE REVIEW

File: website/blog/sql-magic/2021-11-22-sql-surrogate-keys.md [hunk 1]
```diff
@@ -49,21 +49,12 @@ Turns out this is a relatively well-solved problem. To create a surrogate key, y
...
-Let’s take a look at how generating surrogate keys specifically looks in practice across data warehouses, and how you can use one simple dbt macro ([dbt_utils.surrogate_key](https://github.com/dbt-labs/dbt-utils#surrogate_key-source)) to abstract away the null value problem.
-
-
 #### Surrogate keys in BigQuery, Databricks, Redshift and Snowflake
 
 [BigQuery](https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#concat), [Redshift](https://docs.aws.amazon.com/redshift/latest/dg/r_CONCAT.html) and [Snowflake’s](https://docs.snowflake.com/en/sql-reference/functions/concat.html) concat functions returns null if any of the referenced columns for that row returns a null, so to create a proper surrogate key you’d need to wrap each column in a `coalesce` before hashing with an md5 function:
 
 ```sql
-md5 ( concat ( coalesce(column1, ‘’), coalesce(column2, ‘’) ) )
+md5 ( concat ( coalesce(column1, ''), coalesce(column2, '') ) )
```
Comment (patkearns10) [inline website/blog/sql-magic/2021-11-22-sql-surrogate-keys.md]: comment: these were backticks before -- updating to single quotes

**Comment by patkearns10** [comment]
I just noticed this hanging around in here from months ago! @krevitt

**Comment by krevitt** [comment]
> I just noticed this hanging around in here from months ago! @krevitt

Thanks Pat!  Will get this merged in

**Review by krevitt — APPROVED** [review]
lgtm!

**Pull request merged**